### DISCUSSION TITLE
Setting staleTime on next js server

### DISCUSSION THREAD

**Discussion by Zerebokep** [TanStack/query#6104] [Q&A] [answered]
I'm using the dehydration method as explained [here](https://tanstack.com/query/v4/docs/react/guides/ssr) and now I'm wondering if it makes any sense to set a staleTime for the queries which run on the server or does the whole query gets deleted after the request was served?

**Comment by TkDodo** [comment, marked as answer]
when rendering on the server, you create a `new QueryClient()` for every request, so there is no cached data. setting `staleTime` will likely not do much.
